Engine/src/Engine/ResourceManagement/Serializer.cpp

```diff
@@ -191,7 +194,6 @@ namespace Engine
 			j["texWidthFraction"] = c.texWidthFraction;
 			j["texHeightFraction"] = c.texHeightFraction;
 			j["texName"] = c.texName;
-			j["numSprites"] = c.spritesOnSheet;
```

Comment:
Why was this removed? Just because it's regenerated? Because if so, there are probably a few other variables we can remove.

Reply:
I didn't mean to remove this. added back in